problems/4.arrays.js

```diff
@@ -2,25 +2,36 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  if(arr.length < 2){
+    return null;
+  } else {
+    return (arr[0]+ arr [arr.length -1]) / 2;
+  }
 }
-
 /** Duplicates all values inside an array.
 * Sample input: [1,4,2,5]
 * Sample output: [1,4,2,5,1,4,2,5]
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+function duplicateArray(arr) {
+  let arr2 = arr.slice();
+  return arr.concat(arr2);
```

Comment:
I like this solution and love that you wrote arr2 as a variable instead of just throwing the slice into concat